docker/docker-compose-cockroach.yml

```diff
@@ -30,7 +30,7 @@ services:
     - cockroach:postgres
     
   temporal-web:
-    image: temporalio/web:0.23.0
+    image: temporalio/web:0.23.2
```

Comment:
just to double-check  -- is it intentional that temporalio/web's tag is different from all the other images?

Reply:
Yes it is! They are versioned separately unfortunately